src/main/java/edu/ucar/unidata/rosetta/converters/XlsToCsv.java

```diff
@@ -81,22 +82,17 @@ public static boolean convert(String xlsFile, String csvFile) {
 
             //File to store data in form of CSV
             if (csvFile == null) {
-                if (xlsFile.contains(".xlsx")) {
-                    csvFile = xlsFile.replace(".xls", ".csv");
-                } else {
-                    csvFile = xlsFile.replace(".xls", ".csv");
-                }
+                csvFile = FilenameUtils.removeExtension(xlsFile) + ".csv";
             }
 
             File f = new File(csvFile);
-            // if file doesnt exists, then create it
+            // if file doesn't exists, then create it
             if (!f.exists()) {
                 f.createNewFile();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Do something with the "boolean" value returned by "createNewFile". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS899)